helm-values/traction/values-production.yaml

```diff
@@ -21,10 +57,39 @@ tenant_proxy:
         network.openshift.io/policy-group: ingress
   autoscaling:
     enabled: true
-    minReplicas: 4
-    maxReplicas: 8
+    minReplicas: 3
+    maxReplicas: 5
     targetCPUUtilizationPercentage: 80
     targetMemoryUtilizationPercentage: 80
+ui:
+  showOIDCReservationLogin: true
+  oidc:
+    active: true
+    showInnkeeperAdminLogin: true
+    authority: https://loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm
+    jwksUri: https://loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm/protocol/openid-connect/certs
+  ariesDetails:
+    ledgerDescription: "bcovrin-test"
```

Comment:
Can set this to `candy-prod` so it displays in the UI but I actually need to revisit this field now that it's multi-ledger (doing that in my current Traction issue)

Reply:
updated